osu.Game/Beatmaps/Formats/LegacyStoryboardDecoder.cs

```diff
@@ -63,49 +63,42 @@ protected override void ParseLine(Storyboard storyboard, Section section, string
             base.ParseLine(storyboard, section, line);
         }
 
-        private void handleGeneral(Storyboard storyboard, string line)
+        private void handleGeneral(Storyboard storyboard, ReadOnlySpan<char> line)
         {
             var pair = SplitKeyVal(line);
 
             switch (pair.Key)
             {
                 case "UseSkinSprites":
-                    storyboard.UseSkinSprites = pair.Value == "1";
+                    storyboard.UseSkinSprites = pair.Value.SequenceEqual("1");
```

Comment:
Is there a reason for not using `==` as it was originally here?

Reply:
`==` on span means reference equality. It's a pitfall that I found in the previous PR. I was introducing a banned api for `{ReadOnly}Span.op_Equality`. There's also discussion on dotnet runtime to obsolete it.

Reply:
Oh actually `is "1"` is supported in this case, which looks somehow better I think.

Reply:
I...think I would settle with an explicit `SequenceEqual` call over a keyword that is used for type-pattern checking.

Reply:
> a keyword that is used for type-pattern checking

It's able to check many patterns, including my favorite `is Enum.CaseA or Enum.CaseB`, and `is { Prop1.NestedProp: 42 }`. Type-pattern checking is actually less used.